### PR TITLE
Stress: Log additional data to debug stress issues

### PR DESCRIPTION
**Pull request by anthony-murphy**
Seeing some odd nacks in stress that are hard to understand. This additional data may help.

### CODE REVIEW

File: packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts [hunk 1]
```diff
@@ -621,14 +621,18 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 						clientIdOrDocumentId === this.documentId ||
 						clientIdOrDocumentId === this.clientId;
 					const { code, type, message, retryAfter } = nacks[0]?.content ?? {};
-					this.logger.sendTelemetryEvent({
+					const {clientSequenceNumber, referenceSequenceNumber} = nacks[0].operation ?? {};
+					this.logger.logger.sendTelemetryEvent({
```
Comment (anthony-murphy) [inline packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts:625]:
```suggestion
					this.logger.sendTelemetryEvent({
```

File: packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts [hunk 2]
```diff
@@ -621,6 +621,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 						clientIdOrDocumentId === this.documentId ||
 						clientIdOrDocumentId === this.clientId;
 					const { code, type, message, retryAfter } = nacks[0]?.content ?? {};
+					const {clientSequenceNumber, referenceSequenceNumber} = nacks[0].operation ?? {};
```
Comment (andre4i) [inline packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts:624]: Should this be `nacks[0]?.operation ??` ?

**Review by andre4i — APPROVED** [review]

File: packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts (hunk 2 above)
Comment (anthony-murphy) [reply packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts:624]:
```suggestion
					const {clientSequenceNumber, referenceSequenceNumber} = nacks[0]?.operation ?? {};
```

Comment (anthony-murphy) [reply packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts]: yeah. i hate how array typing assumes indexes are non-null

File: packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts [hunk 3]
```diff
@@ -621,6 +621,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 						clientIdOrDocumentId === this.documentId ||
 						clientIdOrDocumentId === this.clientId;
 					const { code, type, message, retryAfter } = nacks[0]?.content ?? {};
+					const {clientSequenceNumber, referenceSequenceNumber} = nacks[0]?.operation ?? {};
```
Comment (anthony-murphy) [inline packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts:624]:
```suggestion
					const { clientSequenceNumber, referenceSequenceNumber } = nacks[0]?.operation ?? {};
```

**Pull request merged**